brainscore_vision/data/geirhos2021/test.py

```diff
@@ -236,7 +236,7 @@ def test_stimulus_set_exist(self, identifier):
         full_name = f"Geirhos2021_{identifier}"
         stimulus_set = load_stimulus_set(full_name)
         assert stimulus_set is not None
-        assert stimulus_set.identifier == full_name
+        assert stimulus_set.identifier == f"{full_name}"
```

Comment:
this change does not seem to do anything.. `full_name == f"{full_name}"` unless `full_name` is not a string (which it should be since it's defined as one a few lines above)

Reply:
Ah changes were mad at a point when master had:  `assert stimulus_set.identifier == f"brendel.{full_name}"`.
This must have changed in a recent update, I will keep as `full_name`.